addons/editor-dark-mode/addon.json

```diff
@@ -962,6 +962,34 @@
         "darkScrollbars": true
       }
     },
+    {
+      "name": "TurboWarp Dark",
+      "id": "tw-dark",
+      "description": "A dark theme based on TurboWarp's dark mode.",
+      "values": {
+        "page": "#111111",
+        "primary": "#4d97ff",
+        "highlightText": "#4d97ff",
+        "menuBar": "#333333",
+        "activeTab": "#1e1e1e",
+        "tab": "#2e2e2e",
+        "selector": "#1e1e1e",
+        "selector2": "#2e2e2e",
+        "selectorSelection": "#111111",
+        "accent": "#111111",
+        "input": "#1e1e1e",
+        "workspace": "#1e1e1e",
+        "categoryMenu": "#111111",
+        "palette": "#111111",
+        "fullscreen": "#111111",
+        "stageHeader": "#111111",
+        "border": "#00000026",
+        "textShadow": false,
+        "dots": true,
+        "darkComments": false,
+        "darkScrollbars": false
```

Comment:
TurboWarp itself doesn't make comments and scrollbars dark, but any dark preset in this addon should.

Reply:
OK, I don't like the way dark comments look, but I could always just enable the preset and turn them off myself.